src/components/shared/ButtonWithPanel/ArrowPanel.css

```diff
@@ -19,11 +19,11 @@
   top: var(--internal-offset-from-top);
   right: calc(var(--internal-offset-from-right) * -1);
   min-width: var(--internal-width);
-  background: hsla(0, 0%, 97%, 0.95);
+  background: hsl(0deg 0% 97% / 0.95);
```

Comment:
The `0deg` part is [hue-degree-notation](https://stylelint.io/user-guide/rules/list/hue-degree-notation/). I'm honestly not sure about that one. I kept it because I thought it's better to be explicit about angles, even for 0, but tell me what you think!

Reply:
Hmm, I can see that how this rule clashes with the `length-zero-no-unit`. I can't say that I liked the new `hue-degree-notation` rule, but I guess it's still okay to keep. I'm not super oppose to it.